iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AreaBoundaryImpl.java

```diff
@@ -19,9 +19,9 @@ public class AreaBoundaryImpl implements AreaBoundary {
 
     final Area area;
 
-    final Terminal terminal;
+    Terminal terminal;
```

Comment:
we could add package protected methods replaceTerminal / replaceBoundary instead, to avoid direct modification of the field from AreaImpl.

Reply:
Done, I switched all attributes to private